cmd/config/internal/commands/tree.go

```diff
@@ -79,9 +80,14 @@ type TreeRunner struct {
 func (r *TreeRunner) runE(c *cobra.Command, args []string) error {
 	var input kio.Reader
 	var root = "."
+	openAPIFileName, err := ext.OpenAPIFileName()
+	if err != nil {
+		return err
+	}
+	matchFilesGlob := append(kio.DefaultMatch, openAPIFileName)
```

Comment:
This might change the underlying slice from `kio.DefaultMatch`, so we should change this to `matchFilesGlob := append([]{openAPIFileName}, kio.DefaultMatch...)`

Reply:
Changed.